### ISSUE TITLE
False Positive ANSIBLE0014 does not allow command:args:stdin

### ISSUE THREAD

**Issue by patrickmacarthur** [willthames/ansible-lint#302]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

```
ansible --version
ansible 2.4.0.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'$HOME/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Aug  2 2017, 11:05:32) [GCC 4.8.5 20150623 (Red Hat 4.8.5-16)]

ansible-lint --version
ansible-lint 3.4.17
```

- ansible installation method: OS package
- ansible-lint installation method: pip

# Desired Behaviour

The `stdin` argument to the `command` module should not trigger the "Environment variables don't work as part of command" error.

# Actual Behaviour (Bug report only)

The EnvVarsInCommandRule (ANSIBLE0014) linter rule rejects the following playbook:

```
- hosts: localhost
  tasks:
    - command: /bin/cat
      args:
        stdin: "Hello, world!"
```

due to the presence of the `stdin` attribute which was added in Ansible 2.4. This appears to be because `stdin` is missing from `expected_args`.
